Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -214,7 +214,7 @@ protected virtual void Awake()
 
         private void OnEnable()
         {
-            if (IsGlobal)
+            if (IsGlobal || !RequiresFocus)
```

Comment:
It's so we can have something that listens to a global voice command, but not a global click or press. I could have one button for menu on a toolbar, which needs to be focused to click, but a global voice command that I can speak at any time "menu" to show the menu. Without these two properties menu would be clicked whenever another button in the scene was clicked.

Reply:
Requires Focus only applies to Speech Commands, I tried to make up a different property name to avoid confusion, though it is confusing :) I'm hoping the context of the Inspector helps.

Reply:
Okay, I understand. This feels potentially confusing to an external developer. Imagine if you're approaching interactable for the first time and you'd see two fields -- requires focus and Is Global. I would be confused for sure. Even as a developer reading the code it's a bit confusing.

Maybe it would help to instead say "speech command requires focus" so that it's clear that this bool is specifically for speech commands.

Reply:
My more general aside about the complexity doesn't need to be addressed in this PR, just giving some feedback that I think we could actually re-use some of the code that already exists in MRTK to make Interactable simpler.